collector/container/konflux-slim.Dockerfile

```diff
@@ -96,44 +85,70 @@ RUN ./builder/install/install-dependencies.sh && \
     ctest -V --test-dir ${CMAKE_BUILD_DIR} && \
     strip -v --strip-unneeded "${CMAKE_BUILD_DIR}/collector/collector"
 
-FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+# Application
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest AS ubi-minimal
+# The installer must be ubi (not minimal) and must be 8.9 or later since the earlier versions complain:
+#  subscription-manager is disabled when running inside a container. Please refer to your host system for subscription management.
```

Comment:
I find this info a bit redundant here but I'm also ok if you keep it.
```suggestion
```

Reply:
https://github.com/stackrox/collector/pull/1573/commits/2eadedead357d172fda01842b7c766b6b9839e11